spec/features/request_to_book_spec.rb

```diff
@@ -1,26 +1,31 @@
 feature 'Request space' do
 
+  before :each do
+    host = db_create_user("dude","d@m.com","dude","dude_pass")
```

Comment:
I am not sure we should have all these actions in a before each, as that can be a source for bugs and would be hard to debug, since you would have to remember all these thinks happen every time. Kind of following the learning from yesterday --> refactor once everything is done.